PocketKit/Sources/Textile/Style/Colors/Palettes.swift

```diff
@@ -23,6 +23,7 @@ public struct UIPalette {
 
     public let homeCellBackground = ColorAsset.ui("homeCellBackground")
     public let saveButtonText = ColorAsset.ui("saveButtonText")
+    public let textfieldURL = UIColor(named: "textfieldURL")
```

Comment:
If we add the color to `Colors.xcassets` in `Textile` and then use `ColorAsset.ui("textfieldURL")`, do we still see the crash? Or only when we call it in line `64` above?

Reply:
That function is currently `internal`, changing it to `public` for this test.

That still crashes the same way.

Reply:
hmm I tried to reproduce the crash using those steps in the previous comment, but wasn't able to.